README.md

```diff
@@ -19,3 +19,78 @@ Github Pages only runs in safe mode, preventing the usage of [symlinks](https://
 1. Copy the contents of `node_modules/@patternfly/patternfly` to `_includes/patternfly`.
 1. Delete `_includes/patternfly/assets` and `_includes/patternfly/icons` as they are not needed.
 1. Commit the changes.
+
+## Deployment
+
+### OKD
+
+1. Copy the generated source in `_site/okd/errors.html` to the corresponding template at https://github.com/openshift/oauth-server/blob/master/pkg/server/errorpage/templates.go.
+1. Copy the generated source in `_site/okd/login.html` to the corresponding template at https://github.com/openshift/oauth-server/blob/master/pkg/server/login/templates.go.
+1. Copy the generated source in `_site/okd/providers.html` to the corresponding template at https://github.com/openshift/oauth-server/blob/master/pkg/server/selectprovider/templates.go.
+1. Submit a pull request to https://github.com/openshift/oauth-server containing the copied changes.
+
+To test the changes:
+
+1. Create a repository in the image registry of your choice to hold the image.
+1. Build image:
+    ```
+    docker build -t quay.io/myaccount/oauth-server:latest -f images/Dockerfile .
+    ```
+1. Push image to image registry:
+    ```
+    docker push quay.io/myaccount/oauth-server:latest
+    ```
+1. Put the console operator in unmanaged state:
+    ```
+    oc patch authentication.operator.openshift.io cluster --patch '{ "spec": { "managementState": "Unmanaged" } }' --type=merge
+    ```
+1. Update the console deployment with the new image:
+    ```
+    oc set image deploy oauth-openshift oauth-openshift=quay.io/myaccount/oauth-server:latest -n openshift-authentication
+    ```
+1. Wait for the changes to rollout:
+    ```
+    oc rollout status -w deploy/oauth-openshift -n openshift-authentication
+    ```
+1. Set branding to OKD:
+    ```
+    oc create -f https://raw.githubusercontent.com/openshift/origin-branding/master/manifests/0000_10_origin-branding_configmap.yaml`
+    ```
```

Comment:
huh, `oc set`, I learn something new every day.

Anyway, these steps are overcomplicated, you can just configure the templates in `oauth/cluster` : `spec.templates`

Reply:
`oc set` is new to me, also

Reply:
`oauth/templates`, interesting: 

https://github.com/openshift/api/blob/master/config/v1/types_oauth.go#L37